pkg/features/versioned_kube_features.go

```diff
@@ -189,6 +189,7 @@ var defaultVersionedKubernetesFeatureGates = map[featuregate.Feature]featuregate
 
 	DRAResourceClaimDeviceStatus: {
 		{Version: version.MustParse("1.32"), Default: false, PreRelease: featuregate.Alpha},
+		{Version: version.MustParse("1.33"), Default: true, PreRelease: featuregate.Beta},
```

Comment:
default true or false? @pohly @johnbelamaric

Reply:
+1 for on by default. Then we don't need to change anything should "DynamicResourceAllocation" go GA with on-by-default: all beta features will become available by default, too.

In the meantime, enabling DRA is easier: users only need to remember to enable the DynamicResourceAllocation feature gate and resource.k8s.io/v1beta1 API and get all beta features automatically, like they they would for other beta feature gates.

Reply:
+1 for on by default from me too, but: If the user enables the DRA APIs, but not the feature gate, what will happen here with this on by default? I guess it won't do any harm, since it just means they will be able to store this but of course there will be no drivers being called, etc. anyway.

Do we check inter-featuregate dependencies anywhere?

Reply:
Jordan at one point suggested that the apiserver should check for "invalid" combinations (DRAAdminAccess on, DynamicResourceAllocation off) and error out. We removed that because it broke the AllApha=true cluster configuration. Such a configuration isn't invalid.

So no, we don't check such dependencies. As you said, enabling DRAResourceClaimDeviceStatus simply has no effect when DynamicResourceAllocation and/or the resource.k8s.io/v1beta1 API are off.